discovery-provider/src/api/v1/users.py

```diff
@@ -669,6 +670,34 @@ def get(self):
         return success_response(users)
 
 
+top_users_response_parser = reqparse.RequestParser()
+top_users_response_parser.add_argument("user_id", required=False)
+top_users_response_parser.add_argument("limit", required=False, type=int)
+top_users_response_parser.add_argument("offset", required=False, type=int)
+top_users_response = make_full_response(
+    "top_users_response", full_ns, fields.List(fields.Nested(user_model_full))
+)
+
+
+@full_ns.route("/top")
+class FullTopUsers(Resource):
+    @full_ns.expect(top_users_response)
+    @full_ns.doc(
+        id="""Get the Top Users""",
```

Comment:
top users by follower count maybe

Reply:
Should I also document that they have to have tracks?

Reply:
probs, just for the future reader :)